### PR TITLE
Add deployments to trial app children

### PR DESCRIPTION
**Pull request by alexweininger**
Add deployments node to trial app children, and ensure that all context menu items are supported for trial apps. Mainly `Open in Portal` must be removed from trial app children.

So the context menu for deployments and deployment items under trial apps now look like:

`DeploymentsTreeItem`:
image

`DeploymentTreeItem`:
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -655,17 +655,17 @@
                 },
                 {
                     "command": "appService.Refresh",
-                    "when": "view == azureAppService && viewItem =~ /^deployments(C|Unc)onnected$/",
+                    "when": "view == azureAppService && viewItem =~ /^deployments(C|Unc)onnected(|NoPortal)$/",
```
Comment (EricJizbaMSFT) [inline package.json:658]: Curious if Nathan agrees, but the "NoPortal" thing in the context value feels a bit overly complex to me. I think I'd rather let it be shown on trial apps and simply throw an error in that case

Comment (alexweininger) [reply package.json:658]: Another alternative: Trial apps don't have the deployments node

Comment (nturinski) [reply package.json:658]: I think I agree with Eric.  Does redeploy work for trial apps?  I think the logs could be useful though.

Comment (alexweininger) [reply package.json:658]: Redeploy works for trial apps.

Comment (EricJizbaMSFT) [reply package.json:658]: Yeah I feel like it's worth it for the deployment logs

Comment (nturinski) [reply package.json:658]: Let's just throw an error then

**Pull request merged**